### ISSUE TITLE
conda-forge package?

### ISSUE THREAD

**Issue by lilyminium** [cmbi/dssp#6]
Thanks for maintaining and updating DSSP, @cbaakman! Would you consider adding a package to conda-forge to make it easier to install?

**Comment by cbaakman** [comment]
Dear @lilyminium, for easy installation I must refer you to the option to build from docker. This requires docker-compose and docker to be installed. 

Build with:
```
docker-compose build
```

Run with:

```
docker-compose run dssp ./mkdssp ...
```

**Comment by lilyminium** [comment]
Thanks @cbaakman, I had a go, another good thing to mention is that you need to mount a directory for the local files to be available. However, I'd rather not use Docker because I'm mostly interested in installing DSSP locally and having conda manage dependencies -- I'm hoping to make a wrapper for it the same way `gmx do_dssp` does, but [for the Python MDAnalysis library](https://github.com/MDAnalysis/mdanalysis/pull/2609), and having the package on conda-forge would make it much easier to integrate. 

I realise you already have a lot on your plate maintaining dssp/hssp/xssp, so if you are alright with DSSP being on conda-forge but not interested in setting up the recipe or maintaining it, I'd be happy to help out with either task :-)

**Comment by cbaakman** [comment]
That's right, you need to mount a directory as volume in docker, in order to work with files outside the container.

@lilyminium you are free to make a conda-forge wrapper and send us a pull request. Just make sure the c++ source code is left intact.

**Comment by lilyminium** [comment]
Thanks @cbaakman! conda-forge recipes are submitted as PRs to conda-forge's [the staged-recipe](https://github.com/conda-forge/staged-recipes/pulls) repo. Would you be happy to maintain it? (the recipe must list a maintainer)

**Comment by cbaakman** [comment]
Yes, you may set my profile name as a maintainer for the dssp recipe.

**Comment by joaomcteixeira** [comment]
Hi,
How has this issue ended up? Is DSSP already deployed on conda-forge?
best,

**Comment by lilyminium** [comment]
Sorry @joaomcteixeira, not yet, it fell off my to-do list for a while. I've opened a PR https://github.com/conda-forge/staged-recipes/pull/11776

**Comment by joaomcteixeira** [comment]
Thanks @lilyminium :+1:
